glide-core/redis-rs/redis/src/cluster_async/mod.rs

```diff
@@ -405,7 +437,7 @@ type ConnectionsContainer<C> =
 
 pub(crate) struct InnerCore<C> {
     pub(crate) conn_lock: RwLock<ConnectionsContainer<C>>,
-    cluster_params: ClusterParams,
+    cluster_params: RwLock<ClusterParams>,
```

Comment:
I remember we have another PR to add rwlock

Reply:
Yes, I saw @barshaul has something, we will need to discuss and aligned.
The PR is not ready, but we will work together above the same core changes, so I opened the current working API. The internal might change.